### DISCUSSION TITLE
why after print xpm install @xpack-dev-tools/riscv-none-elf-gcc@latest --verbose, it says that error: Package '@xpack-dev-tools/riscv-none-elf-gcc@latest' not found

### DISCUSSION THREAD

**Discussion by 1010186930** [xpack-dev-tools/riscv-none-elf-gcc-xpack#35] [Q&A] [answered]
As i mentioned in title, my development environment is WIndows 10. And my xpm version is 0.19.3, then i enter `xpm install @xpack-dev-tools/riscv-none-elf-gcc@latest --verbose` in cmd and it turns out `error: Package '@xpack-dev-tools/riscv-none-elf-gcc@latest' not found`.Did i miss something?

**Comment by 1010186930** [comment]
> As i mentioned in title, my development environment is WIndows 10. And my xpm version is 0.19.3, then i enter `xpm install @xpack-dev-tools/riscv-none-elf-gcc@latest --verbose` in cmd and it turns out `error: Package '@xpack-dev-tools/riscv-none-elf-gcc@latest' not found`.Did i miss something?

![image](https://github.com/user-attachments/assets/fea8d581-e3cf-4680-bcfa-b0ab0b25f83b)

**Reply by ilg-ul** [reply]
Are you behind a firewall, or proxy?

Here is how it should look like:

```
C:\Users\ilg>xpm install @xpack-dev-tools/riscv-none-elf-gcc@latest --verbose
xPack manager - install package(s)

Processing @xpack-dev-tools/riscv-none-elf-gcc@14.2.0-2.1...
Adding to central store 'C:\Users\ilg\AppData\Roaming\xPacks\@xpack-dev-tools\riscv-none-elf-gcc\14.2.0-2.1'...
Downloading https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack/releases/download/v14.2.0-2/xpack-riscv-none-elf-gcc-14.2.0-2-win32-x64.zip...
Extracting 'xpack-riscv-none-elf-gcc-14.2.0-2-win32-x64.zip'...
4545 files extracted in '14.2.0-2.1/.content'
Changing permissions to read-only...

C:\Users\ilg>
```

**Reply by 1010186930** [reply]
> Are you behind a firewall, or proxy?

yes, i am using proxy. and if i turn it off, it shows download fail
![image](https://github.com/user-attachments/assets/f6b4d92c-4c77-4e28-a680-419ea6088c42)
And i tried to download zip in releases, and i print xpm install, but nothing happened?
![image](https://github.com/user-attachments/assets/ddc6dc88-b414-41f1-bb7e-36fdf10bb248)

**Reply by ilg-ul** [reply]
Your antivirus is probably changing the content of the zip and the integrity check fails.

If you managed to download the zip manually, you no longer need to use xpm, just unpack it in a folder of your choice and use it from there.

**Reply by 1010186930** [reply]
> use it from there.
i unzipped it but i find no riscv-gcc stuff
![image](https://github.com/user-attachments/assets/6731d6c0-49d7-4ff3-8a73-5249eb164812)

**Reply by ilg-ul** [reply, marked as answer]
Ah, you downloaded the GitHub project used to build the binaries, you need to download the actual large .zip from the GitHub Releases.

Please read the project documentation carefully for details.

**Reply by 1010186930** [reply]
> Ah, you downloaded the GitHub project used to build the binaries, you need to download the actual large .zip from the GitHub Releases

oh, that is my fault. i didn't notice there is a larger one. i am so careless, thank you
